data/json/mutations/mutations.json

```diff
@@ -7744,5 +7759,40 @@
     "points": 0,
     "description": "You know how to project your voice.  You are able to shout much louder than the average person.",
     "valid": false
+  },
+  {
+    "type": "mutation",
+    "id": "FELINE_FLEXIBILITY",
+    "name": { "str": "Feline Flexibility" },
+    "points": 4,
+    "description": "Your muscles, joints, and movement center of your brain have changed internally somewhat, allowing you to traverse obstacles with ease.",
+    "valid": false,
+    "purifiable": false,
+    "threshreq": [ "THRESH_FELINE" ],
+    "category": [ "FELINE" ],
+    "movecost_obstacle_modifier": 0.5
+  },
+  {
+    "type": "mutation",
+    "id": "HATES_WATER",
```

Comment:
I love this one.  Thanks for adding this.

Reply:
Thanks, I tried.